.github/workflows/pull-request-commentor.yaml

```diff
@@ -0,0 +1,90 @@
+---
+name: Add comment
+# yamllint disable-line rule:truthy
+on:
+  pull_request_target:
+    types:
+      - labeled
```

Comment:
can the `ci/*` branch(es) be excluded from this?

Reply:
also wondering if this can be triggered after a rebase and the label is set...

Reply:
Yes we can trigger only on required branches. for that  we need to remove and re-add the label for now (we can see how that can be automated later?).

Reply:
updated to run on devel and release branch for now